package.json

```diff
@@ -1,12 +1,19 @@
 {
   "name": "snapshot-x",
   "version": "1.0.0",
+  "repository": "snapshot-labs/snapshot-x",
   "license": "MIT",
   "scripts": {
-    "build": "hardhat compile",
+    "compile:l1": "hardhat compile",
+    "compile:l2": "hardhat starknet-compile",
+    "compile": "yarn compile:l1 && yarn compile:l2",
+    "deploy:l2:testnet": "hardhat starknet-deploy --starknet-network alpha",
     "test": "hardhat test",
-    "start": "hardhat node",
```

Comment:
We should have a test for the mock contract on starknet too. That test should live in "test" folder not "scripts"

Reply:
we will, but right now we don't have any tests for our cairo contracts :)

Reply:
There is some test in the scripts/ folder

Reply:
Scott is writing cairo tests currently. the script i wrote isnt really a test